### PR TITLE
 Auto-generate jobs for openshift/origin 

### PR DESCRIPTION
**Pull request by stevekuznetsov**
/assign @smarterclayton

### CODE REVIEW

File: ci-operator/jobs/openshift/origin/openshift-origin-master-postsubmits.yaml [hunk 1]
```diff
@@ -20,7 +20,6 @@ postsubmits:
         - --give-pr-author-access-to-namespace=true
         - --artifact-dir=$(ARTIFACTS)
         - --target=[images]
-        - --target=artifacts
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/origin/openshift-origin-master-postsubmits.yaml]: @smarterclayton what's this all about?

**Comment by smarterclayton** [comment]
lgtm merge at will

**Comment by stevekuznetsov** [comment]
Depends on https://github.com/openshift/ci-operator-prowgen/pull/29

**Pull request merged**